install/primehub-support

```diff
@@ -0,0 +1,407 @@
+	echo "Ceph pg dump log is now logged at $LOG_DIR/ceph_pg_dump_$TIMESTAMP.log"
+	echo "Ceph cluster usage log is now logged at $LOG_DIR/ceph_df_$TIMESTAMP.log"
+	echo "Ceph osd usage log is now logged at $LOG_DIR/ceph_osd_df_tree_$TIMESTAMP.log"
+	echo "Ceph rbd volume list is now logged at $LOG_DIR/ceph_rbd_pool_$TIMESTAMP.log"
+	echo "Ceph rbd device list is now logged at $LOG_DIR/ceph_rbd_device_$TIMESTAMP.log"
+fi
+echo "-------------------------------------------------------------------------"
+echo ""
+
+#=========================================================================#
+
+# Check PrimeHub service status
+echo "-------------------------------------------------------------------------"
+OSD_POOL_NAME=replicapool
+if [[ "$(command -v kubectl-primehub)" == "" ]]; then
+	echo "Kubectl-primehub command not found, skipping....."
+else
+	echo "Kubectl-primehub installed, Check PrimeHub status."
+	kubectl primehub version &> $LOG_DIR/primehub_version_$TIMESTAMP.log
+	#Command usage: kubectl primehub [notebook|job|gitsync|nfs|image-builder|deploy|system] 
+	kubectl primehub notebook &> $LOG_DIR/primehub_notebook_$TIMESTAMP.log
+	kubectl primehub job &> $LOG_DIR/primehub_job_$TIMESTAMP.log
+	kubectl primehub nfs &> $LOG_DIR/primehub_nfs_$TIMESTAMP.log
+	kubectl primehub deploy &> $LOG_DIR/primehub_deploy_$TIMESTAMP.log
+	kubectl primehub system &> $LOG_DIR/primehub_system_$TIMESTAMP.log
+	cd $LOG_DIR
+	kubectl primehub diagnose --path $HOME
+	cd $HOME
+	echo "PrimeHub version log is now logged at $LOG_DIR/primehub_version_$TIMESTAMP.log"
+	echo "PrimeHub notebook list is now logged at $LOG_DIR/primehub_notebook_$TIMESTAMP.log"
+	echo "PrimeHub job list is now logged at $LOG_DIR/primehub_job_$TIMESTAMP.log"
+	echo "PrimeHub nfs list is now logged at $LOG_DIR/primehub_nfs_$TIMESTAMP.log"
+	echo "PrimeHub deploy list is now logged at $LOG_DIR/primehub_deploy_$TIMESTAMP.log"
+	echo "PrimeHub system list is now logged at $LOG_DIR/primehub_system_$TIMESTAMP.log"
+	echo "PrimeHub diagnose log is now stored at $LOG_DIR/primehub-log-$(date +'%Y%m%d%H%M').tgz"
+fi
+echo "-------------------------------------------------------------------------"
+echo ""
+
+#=========================================================================#
+
+# Summary
+FILE_NAME="infuseai-support-log-$(date +'%Y%m%d%H%M')"
+echo "-------------------------------------------------------------------------"
+echo "Compressing logs....."
+cd $LOG_DIR
+tar czf $HOME/$FILE_NAME.tgz * 1> /dev/null 2>&1
+cd $HOME
+echo "Done."
+echo "-------------------------------------------------------------------------"
+echo ""
+
+echo "-------------------------------------------------------------------------"
+echo "Logs folder: $LOG_DIR"
+echo "Log archive: $(realpath $HOME/$FILE_NAME.tgz)"
+echo "Please send the log archive to support engineer for further analysis."
+echo "-------------------------------------------------------------------------"
+
```

Comment:
I think we need to output the information about "OOO command not found"
EX: "Kubectl-primehub command not found."

Reply:
"OOO command not found" is already shown when you run each block of this script.
If we need a "command not found summary" here, sure I can work on that.

Reply:
Create one more log to record "not found" command.